app/src/features/mappings/FhirProfileStep.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+
+import { Icon } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import {
+  Container,
+  Divider,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import AddIcon from "@material-ui/icons/AddCircleOutline";
+import CheckIcon from "@material-ui/icons/Check";
+import clsx from "clsx";
+import { useTranslation } from "react-i18next";
+
+import { Resource } from "services/api/generated/api.generated";
+
+const useStyles = makeStyles((theme) => ({
+  listItem: {
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: 5,
+    boxShadow: `0 1px 5px ${theme.palette.divider}`,
+    marginBlock: theme.spacing(6),
+    display: "flex",
+    alignItems: "center",
+  },
+  icon: {
+    color: theme.palette.text.primary,
+    fill: theme.palette.text.primary,
+    marginRight: theme.spacing(1),
+  },
+  divider: {
+    maxWidth: 200,
+    flexGrow: 1,
+  },
+  divideContainer: {
+    display: "flex",
+    justifyContent: "center",
+  },
+  flameIcon: {
+    fill: "#CC7831",
+    margin: 0,
+  },
+}));
+
+type FhirProfileStepProps = {
+  mapping: Partial<Resource>;
+  onChange?: (mapping: Partial<Resource>) => void;
+};
+
+const FhirProfileStep = ({ mapping }: FhirProfileStepProps): JSX.Element => {
```

Comment:
this is missing the full list of profiles available for this resource because the API is missing the StructureDefinitions ?

Reply:
Yup !
I don't request StructureDefinitions because of this, it's just static string array